chain/consensus/mir/manager.go

```diff
@@ -218,8 +220,13 @@ func (m *Manager) Stop() {
 
 func (m *Manager) SubmitRequests(ctx context.Context, refs []*RequestRef) {
 	for _, r := range refs {
-		err := m.MirNode.SubmitRequest(ctx, r.ClientID, r.ReqNo, r.Hash, []byte{0})
-		if err != nil {
+
+		e := events.NewClientRequests("iss",
+			[]*requestpb.Request{
```

Comment:
Now that the `NewClientRequests` function accepts a whole list of requests for a single event, it would be better to first create a slice of all the client requests and then only create a single `NewClientRequests` event and inject only that one event.

Reply:
fixed